misc/generic_editor.py

```diff
@@ -1,75 +1,36 @@
 # https://github.com/JonathanNickerson/talon_voice_user_scripts
-# TODO tidy this file
 
 import time
 
 import talon.clip as clip
 from talon.voice import Key, press, Str, Context
-from ..utils import (
-    parse_words,
-    join_words,
-    numerals,
-    optional_numerals,
-    is_not_vim,
-    text_to_number,
-)
+from ..utils import parse_words, join_words, numeral_map, is_not_vim, text_to_number
 
 ctx = Context("generic_editor", func=is_not_vim)
-
-
-def jump_to_bol(words):
-    line = text_to_number(words)
-    press("cmd-l")
-    Str(str(line))(None)
-    press("enter")
+numeral_list = sorted(numeral_map.keys())
+ctx.set_list("n", numeral_list)
 
 
 def select_line(m):
```

Comment:
TextEdit. Maybe only? I'm not sure!

Reply:
That's probably right. That can be separated out later I guess, though I think some of these implementations mask the context based commands of same name. I recently commented these out on my branch but hadn't pushed yet because i wasn't sure how/if people were using them.